doc/nrf/ug_nrf5340.rst

```diff
@@ -281,126 +341,111 @@ Follow these steps to program the network sample :ref:`zephyr:bluetooth-hci-rpms
 #. Select the project template, project name, and project location.
 
    * :guilabel:`An externally built executable for Nordic Semiconductor nRF`:
-     This allows specifying the network core hex file created by the build system to be programmed.
+     This template allows specifying the network core HEX file created by the build system to be programmed.
 
-   * :guilabel:`Name`: Specify the name of the project as it will appear in SES. In this example, ``hci_rpmsg_nrf5340_netcore`` is used.
+   * :guilabel:`Name`: Specify the name of the project as it will appear in SES.
+     In this example, we use ``hci_rpmsg_nrf5340_netcore``.
```

Comment:
```suggestion
     This example uses ``hci_rpmsg_nrf5340_netcore``.
```

Reply:
Why?

Reply:
Who is we?

Reply:
Checked RMF and it doesn't like first-person pronouns. I do. But RMF wins. ;)